### ISSUE TITLE
AppPoolDefaults is missing from configuration options

### ISSUE THREAD

**Issue by artisticcheese** [chef-cookbooks/iis#270]
Hello,

Please add option to configure Application Pool Defaults which is set at server level inside <system.applicationHost><applicationPoolDefaults> node.
This is what by default newly created application pool will inherit settings from.

**Comment by EasyAsABC123** [comment]
This will be resolved by 3587cecdd5147a2538aeb1bc36b92026ff8ab580 it will look something like this:

Command Line to set autoStart to true
```
appcmd.exe set config -section:system.applicationHost/applicationPools /applicationPoolDefaults.autoStart:"True" /commit:apphost
```

```ruby
iis_config "Set AutoStart to True by Default" do
  commit "apphost"
  section "system.applicationHost/applicationPools
  property "applicationPoolDefaults.autoStart" => "true"
end
```

in the meantime it can be done (NOT IDEMPOTENT)
```ruby
iis_config "-section:system.applicationHost/applicationPools /applicationPoolDefaults.autoStart:"True" /commit:apphost" do
end
```

**Comment by tas50** [comment]
This has been released
